packages/sanity/src/structure/panes/documentList/DocumentListPaneContent.tsx

```diff
@@ -198,17 +197,7 @@ export function DocumentListPaneContent(props: DocumentListPaneContentProps) {
                   components={{Code: ({children}) => <code>{children}</code>}}
                 />
               </Text>
-
-              {onRetry && (
-                <Box>
-                  <Button
-                    icon={SyncIcon}
-                    onClick={onRetry}
-                    text={t('panes.document-list-pane.error.retry-button.text')}
```

Comment:
Should this string be removed from packages/sanity/src/structure/i18n/resources.ts since it is no longer being used anywhere (as far as I can tell?)

Reply:
Great point! I was a little concerned about removing a string that has already been translated everywhere in favour of something hardcoded 😔. But I think you're right that it's pragmatic to remove it.